src/components/common/markets/clob-widget/clob-market-trade-form.tsx

```
@@ -0,0 +1,419 @@
+
+        if (remainingAmount <= 0) break
+      }
+
+      const averagePrice =
+        totalContracts > 0 ? new BigNumber(totalCost).dividedBy(totalContracts).toNumber() : 0
+      const totalProfit = new BigNumber(totalContracts)
+        .multipliedBy(new BigNumber(1).minus(new BigNumber(averagePrice)))
+        .toNumber()
+
+      return {
+        contracts: isNaN(totalContracts) ? 0 : totalContracts,
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)